spec/features/dishes/show_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+RSpec.describe 'Dish Show Page' do
+  before do 
```

Comment:
I am not familiar with `before do`, is this doing a before(:each) or a before(:all), do you know?